substrate/frame/revive/uapi/src/host.rs

```diff
@@ -561,6 +608,7 @@ pub trait HostFn: private::Sealed {
 	/// - `flags`: Flag used to signal special return conditions to the supervisor. See
 	///   [`ReturnFlags`] for a documentation of the supported flags.
 	/// - `return_value`: The return value buffer.
+	#[cfg(feature = "unstable-api")]
 	fn return_value(flags: ReturnFlags, return_value: &[u8]) -> !;
```

Comment:
This API is not unstable. Without it contracts cannot return anything. Maybe a mishap because it is called `seal_return`.